src/AspNetCoreModuleV2/CommonLib/debugutil.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#include "debugutil.h"
+#include "dbgutil.h"
+
+inline HANDLE g_hStandardOutput;
+
+VOID
+DebugInitialize()
+{
+    g_hStandardOutput = GetStdHandle(STD_OUTPUT_HANDLE);
```

Comment:
Check this behavior in full IIS. This may be an invalid handle on IIS. I'm concerned that this will act weirdly with setting stdout a file later.